junit.js

```diff
@@ -19,33 +23,31 @@ function generateJUnitXML(data, options) {
       return;
     }
     Object.entries(metric.thresholds).forEach(([thresholdName, threshold]) => {
+      const testcaseName = `${escapeHTML(metricName)} - ${escapeHTML(
+        thresholdName,
+      )}`;
+
       if (threshold.ok) {
         cases.push(
-          `<testcase name="${escapeHTML(metricName)} - ${escapeHTML(
-            thresholdName,
-          )}" />`,
+          `<testcase name="${testcaseName}" classname="${classname}" />`,
```

Comment:
Keeping in mind _some_ from pull-request description

>  Some jUnit parsers need classname attribute in order to group the test results.

I'm curious if we should always set a `classname`. I mean, if it's not set (by default), it gets a string `undefined`, is this something we expect?

Reply:
We should, but you're right, `undefined` can lead to confusion, it's better to set it to an empty string.

Reply:
In that case, the class name will get the empty string `''`. I'm unfamiliar with the jUnit parsers, but out of curiosity, have you tested how they parse an empty string?

Depending on :arrow_up: it's either defined as a meaningful `k6-tests` or has a whole `classname="${classname}"` only if the `$classname` is something valid.

What do you think?

Reply:
The `classname` attribute will be ignored if it is missing.
As you can see on the screenshot below, the classname attr doesn't exist, so it defaults to "Unnamed folder"
<img width="996" alt="Screenshot 2023-11-29 at 16 32 48" src="https://github.com/grafana/k6-jslib-summary/assets/114396660/a57e62f6-3684-4cc1-a7cb-7a50372e078b">

In case it exists, it will populate whatever the value is, in this case path.
<img width="1082" alt="Screenshot 2023-11-29 at 16 33 21" src="https://github.com/grafana/k6-jslib-summary/assets/114396660/3ffceca9-5ca2-4f6b-9393-b43df78c7821">

So, we can keep this `Unnamed folder` value, whatever you think is fine.

Reply:
Seems, fine thanks for the input :+1:

Reply:
@olegbespalov updated the PR